### ISSUE TITLE
Launch Essentials Redirects snippets gives warning on WP_HOME and WP_SITEURL defines

### ISSUE THREAD

**Issue by ilovelamp** [pantheon-systems/documentation#2838]
Re: https://pantheon.io/docs/guides/launch/redirects/

## Issue Description

Launch Essentials > Redirect for SEO and Security page includes these lines:

```
  define('WP_SITEURL', $base_url);
  define('WP_HOME', $base_url);
```

This will cause a PHP warning/error as we already have those in our default wp-config.php.

https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L78-L79

```
        define('WP_HOME', $scheme . '://' . $_SERVER['HTTP_HOST']);
        define('WP_SITEURL', $scheme . '://' . $_SERVER['HTTP_HOST']);
```

## Suggested Resolution

Clean up offering on the Launch Essentials > Redirects page to avoid causing warnings.

Remove defines or safely wrap them with defined() calls, such as:

```
if ( !defined( 'WP_HOME' ) {
  define('WP_HOME', $base_url);
}
```

```
if ( !defined( 'WP_SITEURL' ) {
  define('WP_SITEURL', $base_url);
}
```

P.S. typo for word "optoinal" also found in comment in code snippet

**Comment by rachelwhitton** [comment]
Duplicate of #2405

**Comment by alexfornuto** [comment]
Closed by #2853
